schema-engine/sql-introspection-tests/tests/commenting_out/mysql.rs

```diff
@@ -199,6 +209,16 @@ async fn mysql_multi_row_index_warning(api: &mut TestApi) -> TestResult {
           custinfo Json?
         }
 
+        /// This table contains multi-value indices, which are not yet fully supported. Visit https://pris.ly/d/mysql-multi-row-index for more info.
+        model customers_2 {
+          id       BigInt    @id @default(autoincrement())
+          created  DateTime? @default(now()) @db.DateTime(0)
+          modified DateTime? @default(now()) @db.DateTime(0)
+          custinfo Json?
+
+          @@index([created])
```

Comment:
The regular index is showing up in the introspected model, but the unique isn't. I'm not sure I'm reading the [design doc](https://www.notion.so/prismaio/Multi-value-indexes-MySQL-8-1fbf6edcc2bb4d7191b8bd41bdd36ffd) right, but shouldn't both be ignored?